config/allconfig/allconfig.go

```diff
@@ -734,7 +734,7 @@ func (c *Configs) Validate(logger loggers.Logger) error {
 
 // transientErr returns the last transient error found during config compilation.
 func (c *Configs) transientErr() error {
-	for _, l := range c.LanguageConfigSlice {
```

Comment:
`(allconfig.Configs).LanguageConfigSlice` is empty at this point. It is populated by `(allconfig.Configs).Init` which isn't called until _after_ we've checked for transient errors.